bodhi/server/templates/update.html

```diff
@@ -692,6 +692,23 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
+              % if request.registry.settings.get('ci.required', 'False') == 'True':
+              <div class="p-b-1">
+                <div>
+                  % if request.registry.settings.get('ci.url'):
```

Comment:
Should we put the if statement over the ```<div>``` so we don't create an empty div when the setting isn't present?

Also, if this setting isn't defined, will it raise an Exception?